src/Homie/Config.cpp

```diff
@@ -49,6 +47,17 @@ bool Config::load() {
   JsonObject& parsedJson = jsonBuffer.parseObject(buf);
   if (!parsedJson.success()) {
     Interface::get().getLogger() << F("✖ Invalid JSON in the config file") << endl;
+    return JsonObject::invalid();
+  }
+  return parsedJson;
+}
+
+
+bool Config::load() {
+  _valid = false;
+  JsonObject& parsedJson = Config::loadJSONObject();
+  if (!parsedJson.success()) {
```

Comment:
`parsedJson` is freed memory, so behaviour is undefined. (It will work in most cases anyhow, because no one else has overwritten the memory region yet... but technically this is just good luck....)